app/client/src/factory/layer.js

```diff
@@ -338,10 +341,39 @@ export const LayerFactory = {
               station_config: indicatorsStore.state.pt_oev_gueteklasse.config
             };
             promise = ApiService.post_(url, payload, promiseConfig);
-          } else {
+          } else if (lConf.name !== "heatmap_local_accessibility") {
             promise = ApiService.get_(url, promiseConfig);
           }
 
+          if (lConf.name === "heatmap_local_accessibility") {
+            let amenities = {};
+            for (var key in amenityConfiguration) {
+              amenities[key] = {
+                sensitivity: amenityConfiguration[key]["sensitivity"],
+                weight: amenityConfiguration[key]["weight"],
+                max_traveltime: 5
+              };
+            }
+
+            const payload = {
+              mode: "walking",
+              study_area_ids: [91620000],
```

Comment:
We should now have the study area hardcoded here. Do we have it in the datastore.